python/src/aiconfig/schema.py

```diff
@@ -289,7 +289,8 @@ def get_model_name(self, prompt: Union[str, Prompt]) -> str:
             return prompt.metadata.model
         else:
             # Expect a ModelMetadata object
-            return prompt.metadata.model.name
+            prompt_settings = prompt.metadata.model.settings if prompt.metadata.model.settings is not None else {}
+            return prompt_settings.get("model", prompt.metadata.model.name)
```

Comment:
same here

Reply:
Replied to comment above. Reverting this for this diff, but I don't think global settings will work in the UI with the current approach.

model name will be set to something like "AnyscaleEndpoint", and there are no corresponding model settings for it. That's fine for now if all prompt settings are going to be in the prompt itself, but we'll need to figure this out.

Reply:
Hmmm, actually this change is needed

Reply:
For the parsers that have 'model' as a setting right now we already expose 'model' in the settings drawer to be set and that takes precedence over the model.name

Reply:
Ah I see -- so in that case, would we add model_parsers mapping for those models? Or do we expect that registration to have happened outside already?